docs/hooks.md

```diff
@@ -18,15 +18,14 @@ are fully accessible (and modifiable) from within hook implementations. Hence, t
 provides a lot of flexibility to change the EasyBuild functionality should you require it,
 without having to modify the codebase of EasyBuild itself.
 
-
-## Configuring EasyBuild to use your hook implementations
+## Configuring EasyBuild to use your hook implementations {: #some_slug }
```

Comment:
Does this need a genuine slug (e.g. `#hooks_configuration` if we're lifting from the original `.rst`) or are you leaving it as an example?

Reply:
That was my test and accidentally added. Removed in 5c04ec9.